From 0e17e6459c4f59468761ceb99306d869a6fa9c19 Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Thu, 19 Jul 2012 15:14:39 +0800
Subject: ENGR00217128 MX6: support for binding and unbinding console driver

Enable CONFIG_VT_HW_CONSOLE_BINDING to support for binding
and unbinding console driver

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index ffcc997..e032e6c 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1126,7 +1126,7 @@ CONFIG_VT=y
 CONFIG_CONSOLE_TRANSLATIONS=y
 CONFIG_VT_CONSOLE=y
 CONFIG_HW_CONSOLE=y
-# CONFIG_VT_HW_CONSOLE_BINDING is not set
+CONFIG_VT_HW_CONSOLE_BINDING=y
 CONFIG_UNIX98_PTYS=y
 # CONFIG_DEVPTS_MULTIPLE_INSTANCES is not set
 CONFIG_LEGACY_PTYS=y
-- 
1.7.9.5

